src/util/read-package-json.js

```diff
@@ -0,0 +1,7 @@
+import fs from 'fs-promise';
+import path from 'path';
+
+export default (dir) => {
```

Comment:
I'm OK with the JSON conversion being synchronous (it's pretty dam performent anyway 👍